### DISCUSSION TITLE
Hacktoberfest??

### DISCUSSION THREAD

**Discussion by stephannv** [yippee-fun/phlex#274] [Ideas]
Will this repository participate in the hacktoberfest?
I don't even buy shirts, I spend the year with the shirt I get from the event every year.
<img width="602" alt="Screen Shot 2022-10-19 at 18 43 29" src="https://user-images.githubusercontent.com/3025661/196810553-4d93b170-f547-4b25-9dd1-efcbe391e910.png">

**Comment by joeldrapper** [comment]
Do I just need to add the label?

> I don't even buy a shirt, I spend the year with the shirt I get from the event every year.

🤣🤣🤣

**Reply by stephannv** [reply]
Yes, just add `hacktoberfest` label and it's done.

**Comment by stephannv** [comment]
Thanks!!
<img width="557" alt="Screen Shot 2022-10-19 at 21 15 11" src="https://user-images.githubusercontent.com/3025661/196827751-b34ddb6c-f017-4d64-9bdc-9451fa1e2648.png">
